womtool/src/main/scala/womtool/WomtoolMain.scala

```diff
@@ -75,25 +76,50 @@ object WomtoolMain extends App {
   }
 
   def upgrade(workflowSourcePath: String): Termination = {
+    import wdl.model.draft3.elements.ImportElement
+    import wdl.draft2.model.Import
+
+    // Get imports directly from WdlNamespace, because they are erased during WOMification
+    val maybeWdlNamespace: Try[WdlNamespace] =
+      // TODO: do we need directory resolver also? Not clear how to reference it.
```

Comment:
Still TODO? Looks like you already are using a directory resolver?

Reply:
Ah, it's HTTP resolver that's missing